### PR TITLE
Fix drag and drop

### PR DESCRIPTION
**Pull request by fkroockmann**
- Use dnd as plugin
- Rework and split a dnd management

### CODE REVIEW

File: src/tb/apps/content/components/dnd/ContentDrop.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                }
+            },
+
+            /**
+             * Used into a drop event.
+             * Move the content to a new contentset
+             * and set updated old contentset
+             * @param {Object} config
+             */
+            updateContent: function (config) {
```
Comment (ndufreche) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: move this function into DnDManager

File: src/tb/apps/content/components/dnd/ContentDrop.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                }
+            },
+
+            /**
+             * Used into a drop event.
+             * Verify if current contentset accept the content, create the content
+             * from api and show him on the html
+             * @param {Object} config
+             */
+            addContent: function (config) {
```
Comment (ndufreche) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: move this function into DnDManager

File: src/tb/apps/content/components/dnd/ContentDrop.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                this.manager = Manager;
+
+                Core.Mediator.subscribe('on:classcontent:drop', this.onDrop, this);
+            },
+
+            unbindEvents: function () {
+                Core.Mediator.unsubscribe('on:classcontent:drop', this.onDrop);
+            },
+
+            onDrop: function (event) {
```
Comment (ndufreche) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: move this function into ComonDnD

File: src/tb/apps/content/components/dnd/ContentDrop.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+        'use strict';
+
+        return new JS.Class({
+
+            bindEvents: function (Manager) {
+
+                this.manager = Manager;
+
+                Core.Mediator.subscribe('on:classcontent:drop', this.onDrop, this);
```
Comment (ndufreche) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: Bind this on the Manager.

**Comment by ndufreche** [comment]
I think ContentDrop is no more useful, you can remove it and transfer addContent and updateContent into DndManager and onDrop method in ComonDnD

File: src/tb/apps/content/components/dnd/ContentDrop.js (hunk 1 above)
Comment (fkroockmann) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: Why ? she used with a drop :o

File: src/tb/apps/content/components/dnd/ContentDrop.js (hunk 2 above)
Comment (fkroockmann) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: Same

**Comment by fkroockmann** [comment]
I prefere to split the events

File: src/tb/apps/content/components/dnd/ContentDrop.js (hunk 1 above)
Comment (ndufreche) [inline src/tb/apps/content/components/dnd/ContentDrop.js]: Yes but is a method for the content.

**Comment by ndufreche** [comment]
One file for one event, is not very optimal.

**Comment by fkroockmann** [comment]
You can merge :)

**Pull request merged**